src/analysis/raster/qgsrastercalcnode.h

```diff
@@ -125,6 +125,19 @@ class ANALYSIS_EXPORT QgsRasterCalcNode
 
     static QgsRasterCalcNode *parseRasterCalcString( const QString &str, QString &parserErrorMsg ) SIP_FACTORY;
 
+    /**
+     * Returns a List of Strings with raster name
+     * \note since QGIS 3.22
+     */
+    static QStringList referencedLayerNames( const QString &str );
+
+    /**
+     * Returns a List of Strings with a clean version of the raster references
```

Comment:
Returns a list of raster layer names that are refernenced without quotation marks